libs/ui/src/components/CreateArtForm/CreateArtForm.tsx

```diff
@@ -0,0 +1,240 @@
+
+      <Button size="lg" colorScheme="blue" onClick={formDisclosure.onOpen}>
+        <Box mr={{ base: 0, lg: 4 }}>
+          <FaUpload />
+        </Box>
+        <Box display={{ base: 'none', lg: 'block' }}>{t`art.upload`}</Box>
+      </Button>
+
+      <Modal
+        isCentered
+        closeOnOverlayClick={false}
+        isOpen={formDisclosure.isOpen}
+        onClose={formDisclosure.onClose}
+        size={auth?.isLoggedIn ? '4xl' : 'md'}
+      >
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader bg="blue.500" color={'white'}>
+            {t`art.upload`}
+          </ModalHeader>
+          <ModalCloseButton color={'white'} />
+          <ModalBody pos="relative" py={6}>
+            {/* LOADING */}
+            {isLoading && (
+              <Center
+                zIndex={1}
+                pos="absolute"
+                top={0}
+                left={0}
+                boxSize="full"
+                bg="whiteAlpha.900"
+              >
+                <Spinner size="xl" colorScheme="blue" />
+              </Center>
+            )}
+
+            {!auth?.isLoggedIn && (
+              <VStack>
+                <Text>
+                  {t`art.create.require-auth.text`}{' '}
+                  <Navigate href="/user/login" color="blue.500">
+                    {t`art.create.require-auth.button`}
+                  </Navigate>
+                </Text>
+              </VStack>
+            )}
+
+            {/* CREATE FORM */}
+            {auth.isLoggedIn && (
+              <SimpleGrid columns={{ base: 1, lg: 2 }} gap={4}>
+                <FileUploader setImages={setImages} images={images} />
+                <Stack
+                  spacing={4}
+                  as="form"
+                  onSubmit={handleSubmit(handleCreateArt)}
+                >
+                  <FormControl>
+                    <FormLabel
+                      fontSize="sm"
+                      htmlFor="locale"
+                      mb={2}
+                      mt={2}
+                      fontWeight={'600'}
+                    >
+                      {t`language`}
+                    </FormLabel>
+                    <Select
+                      defaultValue={locale}
+                      {...register('locale')}
+                      id="locale"
+                    >
+                      <option value={'en'}>EN (English)</option>
+                      <option value={'nl'}>NL (Nederlands)</option>
+                      <option value={'tr'}>TR (Türkçe)</option>
+                    </Select>
+                  </FormControl>
+                  <FormItem
+                    name="title"
+                    label={t`title`}
+                    isRequired
+                    errors={errors}
+                    register={register}
+                  />
+                  <WSelect
+                    name="categories"
+                    errors={errors}
+                    control={control as any}

```

Comment:
1. We can add a comment.
2. We want to be able to select multiple categories, that's why we need to pass `isMulti` prop

```suggestion
                   // TODO: Investigate why ts complains
                    control={control as any}
                    isMulti
```